1.10.10.dev/alpine3.10/Dockerfile

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright 2019 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.10
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+
+LABEL io.astronomer.docker=true
+LABEL io.astronomer.docker.distro="alpine"
+LABEL io.astronomer.docker.module="airflow"
+LABEL io.astronomer.docker.component="airflow"
+LABEL io.astronomer.docker.airflow.version="1.10.10.dev"
+
+ARG ORG="astronomer"
+ARG VERSION="<1.10.11,>=1.10.10"
```

Comment:
@kaxil what's the reason for this kind of version string?

Reply:
This is so that our dev images are picked up even if it is dev1 or dev2. 

Our current dev images have following structure:

```
astronomer_certified-1.10.10.post1.dev101-py2.py3-none-any.whl
```

Reply:
A new dev package is created when a new commit is added to https://github.com/astronomer/airflow/commits/v1-10-10

Reply:
Oh cool